taxonomies/categories.txt

```diff
@@ -85754,13 +86512,16 @@ ciqual_food_name:fr:Flétan de l'Atlantique ou flétan blanc, cru
 <en:fishes
 en:Atlantic bluefin tuna
 fr:Thon rouge
+ja:タイセイヨウクロマグロ
 ciqual_food_code:en:26069
 ciqual_food_name:en:Atlantic bluefin tuna, raw
 ciqual_food_name:fr:Thon rouge, cru
+# Is this a duplicate?
```

Comment:
Looks like yes. I would suggest to merge both blocks into one:

```
<en:Tunas
en:Atlantic bluefin tunas, atlantic bluefin tuna, northern bluefin tunas, giant bluefin tunas
de:Roter Thun, Großer Thun, Nordatlantischer Thun, Blauflossen-Thunfische
fr:thons rouges, thon rouge
ja:タイセイヨウクロマグロ
la:Thunnus thynnus, Scomber thynnus
nl:Blauwvintonijn
ro:Bucăți de ton
wikidata:en:Q214415
ciqual_food_code:en:26069
ciqual_food_name:en:Atlantic bluefin tuna, raw
ciqual_food_name:fr:Thon rouge, cru```

```suggestion
```

Reply:
[Done!](https://github.com/Naruyoko/openfoodfacts-server/commit/f80c9c14016b62a732337f8103e72c78d98b1fa6)